webui-src/app/people/people_own_contacts.js

```diff
@@ -0,0 +1,111 @@
+const m = require('mithril');
+const rs = require('rswebui');
+const peopleUtil = require('people/people_util');
+
+const contactInfo = () => {
```

Comment:
Is there differences between this component and the one in `people.js`? If not, it would be much better to move this to utils as well. We should try to avoid duplicate code as much as possible.

Reply:
Okk, i will put this component in the people_utils.js